app/core/utils.ozar.coffee

```diff
@@ -304,6 +273,34 @@ else
     progressModalAfter: 1500000 #25 mins
   }
 
+allCourseIDs = _.assign(courseIDs, otherCourseIDs)
+
+courseNumericalStatus = {}
+do ->
+  courseNumericalStatus['NO_ACCESS'] = 0
```

Comment:
if we added a new course and let's say we put the key computer_science_5 and _6, that would cause an issue for old access we gave, correct? 

If we added a Coco course to Oz by re-working it, that would also cause an issue I think.

If we want to assign value this way, I would much rather take values from Array and add a comment that values should be appended to the end of it.

We can't be sure of how an object is looped and it can definitely change internally thus giving a different order on a different machine.

Reply:
oh good point. i need to make sure the course order. also for frontend display. good catch!